web-ui/src/main/resources/catalog/locales/fr-admin.json

```diff
@@ -110,13 +110,13 @@
     "confirmDeleteSchematronCriteriaGroup": "Etes vous certain de vouloir changer le groupe de critères ",
     "contact-updater": "Mise à jour de contact (utilisant une recherche par email)",
     "contactAsXML": "Fragment de XML à insérer pour les contacts correspondant",
-    "contactAsXML-help": "<h5>Exemple:</h5><code class='gn-code'>&lt;gmd:CI_ResponsibleParty xmlns:gco=\"http://www.isotc211.org/2005/gco\" xmlns:gmd=\"http://www.isotc211.org/2005/gmd\"&gt;&lt;gmd:individualName&gt;&lt;gco:CharacterString&gt;A new user&lt;/gco:CharacterString&gt;&lt;/gmd:individualName&gt;&lt;gmd:organisationName&gt;&lt;gco:CharacterString&gt;Organisation&lt;/gco:CharacterString&gt;&lt;/gmd:organisationName&gt;&lt;gmd:contactInfo&gt;&lt;gmd:CI_Contact&gt;&lt;gmd:address&gt;&lt;gmd:CI_Address&gt;&lt;gmd:electronicMailAddress&gt;&lt;gco:CharacterString&gt;newemail@organisation.org&lt;/gco:CharacterString&gt;&lt;/gmd:electronicMailAddress&gt;&lt;/gmd:CI_Address&gt;&lt;/gmd:address&gt;&lt;/gmd:CI_Contact&gt;&lt;/gmd:contactInfo&gt;&lt;gmd:role&gt;&lt;gmd:CI_RoleCode codeListValue=\"pointOfContact\" codeList=\"http://www.isotc211.org/2005/resources/codeList.xml#CI_RoleCode\"/&gt;&lt;/gmd:role&gt;&lt;/gmd:CI_ResponsibleParty&gt;</code>",
```

Comment:
All translations in JSON files should be done in transifex. eg https://www.transifex.com/geonetwork/core-geonetwork/translate/#fr/admin/

Note that you also have search & replace available 

![image](https://user-images.githubusercontent.com/1701393/200281027-1802e3da-ef8f-4d42-b330-fdf163ff7001.png)

(just tested it with nœud)

Reply:
> All translations in JSON files should be done in transifex. eg https://www.transifex.com/geonetwork/core-geonetwork/translate/#fr/admin/

Ok, I will have a look at Transifex. Thanks a lot.
I had a look at the "How to contribute" wiki page but I didn't see any mention to Transifex. Did I miss this information in the docs?

Reply:
https://geonetwork-opensource.org/manuals/4.0.x/en/contributing/translating.html

Reply:
Thanks.